Build failures with mtl-2.3

```
src/Weigh.hs:161:3: error:
    Variable not in scope: unless :: Bool -> IO () -> IO a0
    |
161 |   unless
    |   ^^^^^^
```

As a Hackage trustee I have revised the affected versions on Hackage. See e.g. https://hackage.haskell.org/package/weigh-0.0.16/revisions/.

--- a/src/Weigh.hs
+++ b/src/Weigh.hs
@@ -75,6 +75,7 @@ module Weigh
 import Control.Applicative
 import Control.Arrow
 import Control.DeepSeq
+import Control.Monad (unless)
 import Control.Monad.State
 import Criterion.Measurement
 import qualified Data.Foldable as Foldable

